Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -48,16 +48,21 @@ enum AppSettingsDeeplinkOption {
     }
 }
 
-/// Child settings pages action
-protocol AppSettingsDelegate: AnyObject {
+/// Child settings pages debug action
+protocol DebugSettingsDelegate: AnyObject {
```

Comment:
Changed that to be Debug only, I figured the protocol will grow too much otherwise, so it feels better to have it per settings section instead.